tests/Library/Vanilla/Web/Asset/WebpackAssetProviderTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * @author Adam Charron <adam.c@vanillaforums.com>
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\Library\Vanilla\Web\Asset;
+
+use PHPUnit\Framework\TestCase;
+use org\bovigo\vfs\vfsStream;
+use Vanilla\Web\Asset\LocaleAsset;
+use Vanilla\Web\Asset\WebpackAssetProvider;
+use VanillaTests\Fixtures\MockAddon;
+use VanillaTests\Fixtures\MockAddonProvider;
+use VanillaTests\Fixtures\Request;
+
+/**
+ * Tests for the asset provider.
+ */
+class WebpackAssetProviderTest extends TestCase {
+
+    private $fs;
+
+    /**
+     * @inheritdoc
+     */
+    public function setUp() {
+        $this->fs = vfsStream::setup();
```

Comment:
Is this property necessary? It doesn't look like it's ever referenced.

Reply:
Removed